sleuth_crawler/scraper/scraper/spiders/parsers/generic_page_parser.py

```diff
@@ -1,17 +1,21 @@
 import scrapy
+import re
 from sleuth_crawler.scraper.scraper.spiders.parsers import utils
 from sleuth_crawler.scraper.scraper.items import ScrapyGenericPage
 
-def parse_generic_item(response, children):
+def parse_generic_item(response, children=[]):
```

Comment:
Don't set `children=[]` as a default. I learned today that it doesn't work the way you think it does. See http://docs.python-guide.org/en/latest/writing/gotchas/

Reply:
oh man what the hekkkkkkk